cmd/ingestor/main.go

```diff
@@ -93,6 +96,8 @@ func main() {
 }
 
 func realMain(ctx *cli.Context) error {
+	logger.Infof("%s version:%s", os.Args[0], version.String())
```

Comment:
Would it make sense to move this fmt string into `version.String()` too?

Then all these statements could just be `logger.Infof(version.String())`

Reply:
Sounds good